src/extend/tfoust/contracts.json

```diff
@@ -452,5 +452,7 @@
   "0x849af9533ccb07b857f031688b0ccaf59202abb9",
   "0xf258421052701ac0af75f636475d5f81a4d168a6",
   "0xece092683a7f7e976752e6055a7dea50688a8814",
-  "0xf3f1798eE4D40b78d32A2fE321d045a591E7103a"
+  "0xf3f1798eE4D40b78d32A2fE321d045a591E7103a",
```

Comment:
@zSlammin  while you have a PR up here probably worth making this lower case since i believe that is how reservoir needs their collection ids